components/ContactCard/utils.ts

```diff
@@ -18,3 +18,11 @@ export function getSocialHandles(contactInfo: ContactInfo) {
         twitter: isValidTwitter ? socialLinks.getProfileId('twitter', twitter) : null,
     };
 }
+
+export function getUrl(url: string): URL | null {
```

Comment:
I remember adding URL normalization to content renderer for URLs without protocol. It should be applied as a transform brefore the render step, if I remember correctly. Has it been broken? 🤔

Reply:
I am not sure but I've noticed that the URL object is not being created correctly for some Prezly newsrooms so I've used the try/catch in the content renderer and migrated it here just in case.